utils/discounts/evergreen.ts

```diff
@@ -0,0 +1,25 @@
+export const renewalUpsell: Upsell = {
+  duration: 3,
+  paidDuration: 2,
+  maxDuration: 1,
+  discountId: "0",
+  imageUrl: "/register/registerUpsell.webp",
+  title: {
+    desc: "Unlock Extended Domain",
+    catch: "3 Years for the Price of 2!",
+  },
+  desc: "Don't miss out on this one-time offer! This is your chance to secure extended benefits and ensure a lasting digital presence.",
+};
+
+export const registrationUpsell: Upsell = {
+  duration: 3,
+  paidDuration: 2,
+  maxDuration: 1,
+  discountId: "0",
+  imageUrl: "/register/registerUpsell.webp",
+  title: {
+    desc: "Unlock Extended Domain",
+    catch: "3 Years for the Price of 2!",
+  },
+  desc: "Don't miss out on this one-time offer! This is your chance to secure extended benefits and ensure a lasting digital presence.",
+};
```

Comment:
Here we could use a file structure like we do in calls like so:
```ts
const evergreenDiscounts = {
  registrationUpsell,
  renewalUpsell
};

export default evergreenDiscounts;
```

When we read we would directly understand this, the con is it would be very long `evergreenDiscounts.registrationUpsell`. 

We would delete `Upsell` at the end maybe? I'll let you decide.

Reply:
Yes I deleted the upsell at the end so it's shorter